src/external/miniaudio.h

```diff
@@ -36078,9 +36078,15 @@ static ma_result ma_context_get_device_info_from_fd__audio4(ma_context* pContext
         ma_uint32 channels;
         ma_uint32 sampleRate;
 
+#ifdef __NetBSD__
```

Comment:
Actually those changes should be on miniaudio side because they will be ovewritten when updating miniaudio...

Reply:
already merged into dev branch of miniaudio, so it is for until we get new release